layouts/case-studies/list.html

```diff
@@ -11,4 +11,8 @@
   
 </section>
   </div>
+{{ end }}
+
+{{ define "appcode" }}
```

Comment:
Loading this script on every page didn't feel like the best idea. Defining an `appcode` block that's separate from the `scripts` loaded on every page felt like a better approach. This will also make it easier to separate code by page going forward.